src/components/SectionHeader/SectionHeader.tsx

```diff
@@ -3,29 +3,32 @@ import styled from 'styled-components/macro';
 
 import { fontSizes } from 'styles';
 
+type ElementProps = {
+  noPadding?: boolean;
+};
```

Comment:
Maybe we can standardize styled-component prop types as `StyleProps` or `StyledProps`

Reply:
Hmmm, let's chat about this. I'm open to pretty much anything but this project currently uses `ElementProps` everywhere, so we can decide on a name and then change everything out later.